tests/acceptance/features/apiGraph/getUser.feature

```diff
@@ -1005,3 +1005,223 @@ Feature: get users
+  Scenario Outline: admin user gets the drive information of a user with different user role
+    Given the administrator has given "Alice" the role "Admin" using the settings api
+    And the administrator has given "Brian" the role "<userRole>" using the settings api
+    When the user "Alice" gets user "Brian" along with his personal drive information using Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+    """
+    {
+      "type": "object",
+      "required": [
+        "driveAlias",
+        "driveType",
+        "id",
+        "name",
+        "webUrl",
+        "owner",
+        "quota",
+        "root"
+      ],
+      "properties": {
+        "driveAlias": {
+          "type": "string",
+          "enum": ["personal/brian"]
+        },
+        "driveType": {
+          "type": "string",
+          "enum": ["personal"]
+        },
+        "id": {
+          "type": "string",
+          "pattern": "^%space_id_pattern%$"
+        },
+        "name": {
+          "type": "string",
+          "enum": ["Brian Murphy"]
+        },
+        "webUrl": {
+          "type": "string",
+          "pattern": "^%base_url%/f/%space_id_pattern%$"
+        },
+        "owner": {
+          "type": "object",
+          "required": [
+            "user"
+          ],
+          "properties": {
+            "user": {
+              "type": "object",
+              "required": [
+                "displayName",
+                "id"
+              ],
+              "properties": {
+                "displayName": {
+                  "type": "string",
+                  "enum": [""]
+                },
+                "id": {
+                  "type": "string",
+                  "pattern": "^%user_id_pattern%$"
+                }
+              }
+            }
+          }
+        },
+        "qouta": {
+          "type": "object",
+          "required": [
+            "state"
+          ],
+          "properties": {
+            "state": {
+              "type": "string",
+              "enum": ["normal"]
+            }
+          }
+        },
+        "root": {
+          "type": "object",
+          "required": [
+            "webDavUrl"
+          ],
+          "properties": {
+            "webDavUrl": {
+              "type": "string",
+              "pattern": "^%base_url%/dav/spaces/%space_id_pattern%$"
+            }
+          }
+        }
+      }
+    }
+    """
+    Examples:
+      | userRole    |
+      | Admin       |
+      | Space Admin |
+      | User        |
+      | Guest       |
+
+
+  Scenario Outline: non-admin user tries to get drive information of other user with different user role
+    Given the administrator has given "Alice" the role "<user-role>" using the settings api
+    And the administrator has given "Brian" the role "<userRole>" using the settings api
+    When the user "Alice" gets user "Brian" along with his personal drive information using Graph API
```

Comment:
if i am not wrong the API endpoint only retrieves the personal drive information right?  if so then the step name and the scenario name can be adjust accordingly.